program.cpp

```diff
@@ -5,16 +5,19 @@ using std::vector;
 
 int main()
 {
+    
     // Load all resources
     load_resource_bundle("bundle", "resources.txt");
-
+    Option option=Option();
+    bool hasBackgroundMusic=false;
+    Audio *audio=new Audio();
     // Instantiate introductory classes
     Helper helper;
     ConfigData configData;
     Splashscreen intro_splashkit("intro_splashkit");
 
     // Open window and toggle border off.
-    open_window("arcade-machine", 1920, 1080);
+    open_window("arcade-machine", 1500, 1080);
```

Comment:
Hi @huydnnk97, The window must be 1920 x 1080, so this change will need to revert. I would suggest making the image you are trying to accommodate for, comply with these dimensions.